derive/src/object.rs

```diff
@@ -374,7 +374,12 @@ pub fn generate(
                         }
                         None => quote! { ::std::option::Option::None },
                     };
-                    let param_getter_name = get_param_getter_ident(&ident.ident.to_string());
+                    // We're generating a new identifier, so
+                    // ignore the prefix for raw idents
+                    let param_name = ident.ident.to_string();
+                    let param_name = param_name.strip_prefix("r#").unwrap_or(&param_name);
```

Comment:
It is better to use the `unraw` function. 😁

```rust
let param_name = ident.ident.unraw().to_string();
```

Reply:
I had no idea this method existed - thanks for letting me know :smile: